index.html

```diff
@@ -0,0 +1,813 @@
+                                <svg class="icon icon-behance">
+                                    <use xlink:href="#icon-behance"></use>
+                                </svg>
+                            </div>
+                        </div>
+                    </div>
+                    <h2 class="team-member__name">Jenny L Martian</h2>
+                    <h3 class="team-member__job">UX Designer</h3>
+                </div>
+                <div class="team-member">
+                    <div class="team-image__container">
+                        <img src="images/team_3.jpg" alt="Johnathan">
+                        <div class="team-contact__icons">
+                            <div class="clearfix pbs">
+                                <svg class="icon icon-facebook">
+                                    <use xlink:href="#icon-facebook"></use>
+                                </svg>
+                            </div>
+                            <div class="clearfix pbs">
+                                <svg class="icon icon-twitter">
+                                    <use xlink:href="#icon-twitter"></use>
+                                </svg>
+                            </div>
+                            <div class="clearfix pbs">
+                                <svg class="icon icon-dribbble">
+                                    <use xlink:href="#icon-dribbble"></use>
+                                </svg>
+                            </div>
+                            <div class="clearfix pbs">
+                                <svg class="icon icon-behance">
+                                    <use xlink:href="#icon-behance"></use>
+                                </svg>
+                            </div>
+                        </div>
+                    </div>
+                    <h2 class="team-member__name">James David Doe</h2>
+                    <h3 class="team-member__job">Creative Director</h3>
+                </div>
+            </div>
+        </section>
+        <section id="portafolio" class="section-gray">
+            <div class="text-center">
+                <h1>Lastest Work</h1>
+                <p> Lorem ipsum dolor sit amet, maecenas eget vestibulum
+                    justo
+                    imperdiet</p>
+            </div>
+
+            <div class="portafolio-images">
+                <div class="team-image__container">
+                    <img src="images/portfolio-img1.jpg" alt="Johnathan">
+                    <div class="portafolio-ilustration">
+                        <div class="portafolio-ilustration__content">
+                            <div class="portafolio-icon">
+                                <svg class="icon icon-link">
+                                    <use xlink:href="#icon-link"></use>
+                                </svg>
+                            </div>
+                            <h1>Ilustration</h1>
```

Comment:
the whole page must have only one h1, the rest of the headers must be used to have a map of the page, they are not meant to be used to give styles to the content